cloudamqp/resource_cloudamqp_aws_eventbridge.go

```diff
@@ -0,0 +1,115 @@
+package cloudamqp
+
+import (
+	"github.com/84codes/go-api/api"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAwsEventBridge() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEventBridgeCreate,
+		Read:   resourceAwsEventBridgeRead,
+		Delete: resourceAwsEventBridgeDelete,
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:        schema.TypeInt,
+				ForceNew:    true,
+				Required:    true,
+				Description: "Instance identifier",
+			},
+			"aws_account_id": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: "The 12 digit AWS Account ID where you want the events to be sent to.",
+			},
+			"aws_region": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: "The AWS region where you the events to be sent to. (e.g. us-west-1, us-west-2, ..., etc.)",
+			},
+			"vhost": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: "The VHost the queue resides in.",
+			},
+			"queue": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: "A (durable) queue on your RabbitMQ instance.",
+			},
+			"with_headers": {
+				Type:        schema.TypeBool,
+				ForceNew:    true,
+				Required:    true,
+				Description: "Include message headers in the event data.",
+			},
+			"status": {
```

Comment:
Exclude? During my testing, this never got updated in our backend and always was null.

Reply:
No, I think we should keep it, looks like it [can have a value](https://github.com/84codes/cloudamqp-eventbridge/search?q=status) and I think it could be useful if someone refreshes the state while troubleshooting.